editor/plugins/skeleton_editor_plugin.cpp

```diff
@@ -114,7 +114,8 @@ PhysicalBone *SkeletonEditor::create_physical_bone(int bone_id, int bone_child_i
 	bone_shape->set_shape(bone_shape_capsule);
 
 	Transform body_transform;
-	body_transform.origin = Vector3(0, 0, -half_height);
+	body_transform.origin = Vector3(0, half_height, 0);
+	body_transform.basis = body_transform.rotated(body_transform.basis[0], 3.141593 / 2.0).basis;
```

Comment:
We have `MATH_PI`, the value should not be hardcoded with low precision.

Reply:
Sounds good, didn't know about that const. Should I just commit and push?

Reply:
Commit with `--amend`, so it stays as only one commit. Then push with `--force`.

Reply:
Ok this should be fixed using the `Math_PI` const now